src/pages/Home.scss

```diff
@@ -0,0 +1,4 @@
+div{

```

Comment:
Gotcha, you're using that div, so ignore [this comment](https://github.com/MarcelloAragoni/mentor/pull/1/files#r843855099).

Right, we have 2 points here:

1. Every div inside the `Home.jsx` and its children will be orange (the div is too generic)
2. We always named our styles selectors. So instead of using `div` as a selector, you should use something that represents this div, that could be `home`


References:
- https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Selectors